TeamCode/src/test/java/com/acmerobotics/relicrecovery/path/LinearPathExecutorTest.java

```diff
@@ -0,0 +1,37 @@
+package com.acmerobotics.relicrecovery.path;
+
+import com.acmerobotics.relicrecovery.drive.MecanumDrive;
+import com.acmerobotics.relicrecovery.localization.Vector2d;
+
+import org.junit.Test;
+import org.mockito.InOrder;
+
+import java.util.Arrays;
+
+import static org.mockito.Mockito.*;
+
+/**
+ * @author Ryan
+ */
+
+public class LinearPathExecutorTest {
+
+    @Test
+    public void testBasicPathExecution() {
```

Comment:
I think Mockito provides a nice and clean way to test this kind of code. Might be worth taking a look at.